### ISSUE TITLE
Update voting results / vote weights after each delegation

### ISSUE THREAD

**Issue by oliverbarnes** [liquidvotingio/api#52]
Right now results and weights only get updated on vote creation/deletion and delegation deletion for a specific proposal. There's an issue for recalculating on deletion of global delegations, but we also need to recalculate on delegation creations, both proposal-specific and global

**Comment by niklaslong** [comment]
I think we should indiscriminately update past votes for now and in future work out whether we want a voting period (#65)?

**Comment by oliverbarnes** [comment]
Yeah, makes sense. Voting period (or result freeze) is definitely a separate issue and can of worms
